pkg/features/kube_features.go

```diff
@@ -597,8 +600,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CPUManager:                     {Default: true, PreRelease: featuregate.Beta},
 	CPUCFSQuotaPeriod:              {Default: false, PreRelease: featuregate.Alpha},
 	TopologyManager:                {Default: true, PreRelease: featuregate.Beta},
-	ServiceNodeExclusion:           {Default: false, PreRelease: featuregate.Alpha},
-	NodeDisruptionExclusion:        {Default: false, PreRelease: featuregate.Alpha},
+	ServiceNodeExclusion:           {Default: true, PreRelease: featuregate.Beta},
```

Comment:
it is super confusing that we don't reference `ServiceNodeExclusion` anywhere, but instead redefine the constant in `serviceNodeExclusionFeature`

Reply:
They did that to cut the linkage betwene packages, so i agree both with your confusion and wiht the motivation for the cut